examples/java/pom.xml

```diff
@@ -327,24 +327,21 @@
   </profiles>
 
   <build>
-    <pluginManagement>
-      <plugins>
-        <!-- BEAM-933 -->
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>findbugs-maven-plugin</artifactId>
-          <configuration>
-            <skip>true</skip>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
 
+      <!-- BEAM-933 -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+
       <plugin>
```

Comment:
My rational is that pluginManagement is intended to configure project builds that inherit from this one (it creates a context) but in the files where I changed this, those are 'leaf' poms not parents so this is not needed. I agree that this can be a subtle difference but my motivation was also to get rid of the warnings that this produced on unsetted version for 'fake parents'.